content/blog/2021/containerqueries.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Getting Started with Container Queries
+sub: Now is the time to begin experimenting with a long requested layout tool. 
+author: davidh
+date: 2021-04-02
+image:
+  src: blog/2021/widequote.jpg
+  alt: blockquote with magenta background and polygonal clip-path
+tags:
+  - CSS
+  - CSSWG
+  - Article
+  - Container Queries
+summary: |
+  The `@container` query, that elusive feature developers have been requesting and proposing for years, has finally made its debut in browsers. Well, sort of. What are **container queries** exactly and how do they work?
+---
+
+{% import 'embed.macros.njk' as embed %}
+
+  In the latest version of [Chrome Canary](https://www.google.com/chrome/canary), the most recent `@container` query proprosal, developed by OddBird's own [Miriam Suzanne](https://www.oddbird.net/authors/miriam/), is available for use behind an experimental flag. Miriam's [proposal](https://github.com/w3c/csswg-drafts/issues/5796) builds upon the ideas previously proposed by others, in particular David Baron, who wrote the [original draft](https://github.com/dbaron/container-queries-implementability). 
+
+
+ ## Background
+
+`@Media` queries sparked a responsive design revolution by allowing authors to change the style of elements based on the size of the entire viewport. Up until now, what could not be done was changing the style of an element based on the size of its parent element. 
+
+## Syntax and an example
+
+ The proposed `@container` query syntax feels a lot like writing a `@media` query, but a main difference is that a `@container` query has to be implemented in two parts.
+
+In this demo (which only works in the latest version of [Chrome Canary](https://www.google.com/chrome/canary)), a `blockquote` is styled differently depending on the size of its container. 
+
+{{ embed.codepen(
+  id='YzpywrZ',
```

Comment:
Quick thought on the pen itself: If the overall layout used a media-query, the fallback would show all three quotes in differently-sized columns. That might make the issue more clear for people not on Canary?

Reply:
Makes sense to separate the overall layout into a media query. Made that change in the CodePen. I think that helps to clear this up?